test/index.spec.ts

```diff
@@ -12,6 +12,10 @@ describe('resin-semver', () => {
 			});
 		});
 
+		it('should correctly sort lists of versions', () => {
+			expect(versions.sort(semver.compare)).to.eql(versions);
```

Comment:
That's always true since `sort` works [in memory](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Return_value) and returns the same reference.
```js
x=[3,2,1];
x.sort() === x; // => true
```
How about something like:
```js
expect(versions.slice().sort(semver.compare)).to.eql(versions);
```
?